mico-core/src/main/java/io/github/ust/mico/core/model/MicoEnvironmentVariable.java

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
There are two classes (`MicoEnvironmentVariable`, `MicoLabel`) consisting only of a Key-Value pair. Can't we just use one of them for both?

Reply:
No. They are different. For example the `MicoEnvironmentVariable` has the properties `name` and `value` and the `MicoLabel` has the properties `key` and `value`. Furthermore they require different annotation e.g. for validation.